ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -377,6 +377,25 @@ public void start() throws ReplicationException {
   }
 
   /**
+   * Enables/disables {@code types} of replication on the given {@code id}s.
+   * @param ids the {@link PartitionId}s to enable/disable it on.
+   * @param origins the list of datacenters from which replication should be enabled/disabled.
+   * @param enable whether to enable ({@code true}) or disable.
+   */
+  public boolean controlReplicationForPartitions(List<? extends PartitionId> ids, List<String> origins,
```

Comment:
what is the expected behaviour if the node is marked as down and we trigger a admin request to disable/enable replication for the same? I mean, would this method return true or false?